OTAnalytics/plugin_ui/customtkinter_gui/dummy_viewmodel.py

```diff
@@ -574,6 +582,7 @@ def _update_selected_videos(self, videos: list[Video]) -> None:
         self._update_enabled_video_buttons()
 
     def add_video(self) -> None:
+        self.frame_remark.load_remark()  # TODO Change where to place this
```

Comment:
Might be triggered by an otconfig load. Let's check together for the correct place.